backend/siarnaq/api/compete/views.py

```diff
@@ -387,6 +389,59 @@ def scrimmage(self, request, pk=None, *, episode_id):
         serializer = self.get_serializer(page, many=True)
         return self.get_paginated_response(serializer.data)
 
+    def get_historical_rating_ranking(self, episode_id, queryset, limit=None):
```

Comment:
This could probably use a docstring to explain its purpose since it's quite a lengthy boi